discord/bot.py

```diff
@@ -31,16 +37,50 @@ async def bad_button(
         button: discord.ui.Button,
     ):
         await interaction.response.send_message(
-            content="This is an edited button response!"
+            content="Bad desponse!"
         )
+        filename = f"discord/private_chats/{interaction.user.id}.txt"
+        with open(filename, "a") as file:
+            file.write(f"0\n")
+        self.stop()
+
+
+
+
+
+
+import asyncio
+from collections import defaultdict
+async def chats_history(bot, last_messages_per_channel):
+    for guild in bot.guilds:
+        for channel in guild.text_channels:
+            if channel.permissions_for(guild.me).read_messages:
+                async for message in channel.history(limit=100):
+                    last_messages_per_channel[channel.id].append(f"{message.author.name}: {message.content}")
+
+                    
+async def repetitor(bot, last_messages_per_channel):
+    while True:
+        await chats_history(bot, last_messages_per_channel)
+        await asyncio.sleep(30)
+
+async def start_bot(bot):
```

Comment:
We have another way to start our bot on main branch! You will need to resolve your conflicts when marging